CME/pages/CMEQuizPage.php

```diff
@@ -209,6 +314,51 @@ protected function initQuiz()
 	}
 
 	// }}}
+	// {{{ protected function generateQuiz()
+
+	protected function generateQuiz()
+	{
+		$class_name = SwatDBClassMap::get('CMEQuiz');
+
+		$quiz = new $class_name();
+		$quiz->setDatabase($this->app->db);
+
+		$quiz->createdate = new SwatDate();
+		$quiz->createdate->toUTC();
+		$quiz->save();
```

Comment:
No need to do this anymore since we have removed the `account` column from Inquisition.